public/app/themes/justice/inc/search.php

```diff
@@ -195,7 +242,7 @@ public function highlightResults($text)
 
     public function formattedUrl(string $url): string
     {
-
+        return $url;
```

Comment:
This looks like it's been left by mistake since there's a return later in the function.